src/main/java/nextstep/subway/path/domain/Path.java

```diff
@@ -1,19 +1,29 @@
 package nextstep.subway.path.domain;
 
 import java.util.List;
+import nextstep.subway.auth.domain.LoginMember;
 import nextstep.subway.station.domain.Station;
 
 public class Path {
     private final List<Station> stations;
     private final Integer distance;
+    private Integer fare;
 
-    public Path(List<Station> stations, Integer distance) {
+    public Path(List<Station> stations, Integer distance, Integer fare) {
         this.stations = stations;
         this.distance = distance;
+        this.fare = fare;
     }
 
-    public static Path of(List<Station> vertexList, double weight) {
-        return new Path(vertexList, (int) weight);
+    public static Path of(List<Station> vertexList, double weight, Integer lineOverFare) {
+        return new Path(vertexList, (int) weight, lineOverFare);
+    }
+
+    public void calculateFare(LoginMember loginMember) {
+        DistanceFarePolicy distanceFarePolicy = DistanceFarePolicy.findByDistance(distance);
+        fare += distanceFarePolicy.calculate(distance);
+        AgeFarePolicy ageFarePolicy = AgeFarePolicy.findByAge(loginMember.getAge());
+        this.fare = ageFarePolicy.calculate(fare);
```

Comment:
여기는 이렇게 더해주지 않아도 될까요...?
```suggestion
        this.fare += ageFarePolicy.calculate(fare);
```

Reply:
팩토리 메서드 `of()`에서 노선 별 추가요금을 이미 받아와서 객체를 생성하기 때문에 말씀하신 부분에서 더해주지 않으면 노선 별 추가요금이 계산에서 제외됩니다. 😄